Gems/Atom/Tools/AtomToolsFramework/Code/Include/AtomToolsFramework/Util/MaterialPropertyUtil.h

```diff
@@ -41,6 +42,21 @@ namespace AtomToolsFramework
     //! Compare equality of data types and values of editor property stored in AZStd::any
     bool ArePropertyValuesEqual(const AZStd::any& valueA, const AZStd::any& valueB);
 
+    //! Convert the property value into the format that will be stored in the source data
+    //! This is primarily needed to support conversions of special types like enums and images
+    bool ConvertToExportFormat(
+        const AZStd::string& exportPath,
+        const AZ::RPI::MaterialTypeSourceData::PropertyDefinition& propertyDefinition,
+        AZ::RPI::MaterialPropertyValue& propertyValue);
+
+    //! Generate a file path from the exported file to the external reference.
+    //! This function is to support copying or moving a folder containing materials, models, and textures without modifying the files. The
+    //! general case returns a relative path from the export file to the reference file. If the relative path is too different or distant
+    //! from the export path then we return the asset folder relative path. An alternate solution would be to only use export folder
+    //! relative paths if the referenced path is in the same folder or a sub folder the assets are not generally packaged like that.
```

Comment:
I found the placement of the second sentence a bit confusing. Also, IMO the last sentence is best used with the implementation, not as part of the API. Also, documenting the params would be helpful here.
```suggestion
    //! Generate a file path from the exported file to the external reference, using a relative path where appropriate.
    //! The general case returns a relative path from the export file to the reference file. If the relative path is too different or distant
    //! from the export path then we return the asset folder relative path. 
    //! Using relative paths where possible allows easier copying or moving a folder containing materials, models, and textures without modifying the files to fix references.
    //! @param exportPath the path to where the file is being saved
    //! @param referencePath the path to a file being referenced, which may be converted to a relative path
    //! @param maxPathDepth this indicates the maximum number of upward steps ("../") that can be used to form the relative path
    //! @return either a relative path to @referencePath or the original @referencePath if a relative path cannot be formed
```